Easydict/objc/ViewController/View/WordResultView/EZWordResultView.m

```diff
@@ -1276,16 +1278,16 @@ - (void)updateWebViewAllIframeFontSize {
     CGFloat fontSize = Configuration.shared.fontSizeRatio * 100;
 
     NSString *jsCode = [NSString stringWithFormat:
-    @"var iframes = document.querySelectorAll('iframe');"
-    @"for (var i = 0; i < iframes.length; i++) {"
-    @"   var iframe = iframes[i];"
-    @"   var frameDoc = iframe.contentDocument || iframe.contentWindow.document;"
-    @"   frameDoc.body.style.fontSize = '%f%%';"
-    @"};", fontSize];
-    
+                        @"var iframes = document.querySelectorAll('iframe');"
+                        @"for (var i = 0; i < iframes.length; i++) {"
+                        @"   var iframe = iframes[i];"
+                        @"   var frameDoc = iframe.contentDocument || iframe.contentWindow.document;"
+                        @"   frameDoc.body.style.fontSize = '%f%%';"
+                        @"};",
+                        fontSize];
+
     [self evaluateJavaScript:jsCode completionHandler:^(id _Nullable result, NSError *_Nullable error) {
         if (!error) {
-            
```

Comment:
Please have a comment here instead of empty content.

Reply:
This is a unused method, I have remove it.